contracts/Voter.sol

```diff
@@ -12,35 +12,49 @@ import "./interfaces/IRomulusDelegate.sol";
 contract Voter is Ownable {
   using SafeERC20 for IERC20;
 
-  uint8 public immutable support;
   IVotingDelegates public immutable votingToken;
   IRomulusDelegate public immutable romulusDelegate;
+  address public controllerAddress;
 
   constructor(
-    uint8 _support,
     IVotingDelegates _votingToken,
     IRomulusDelegate _romulusDelegate
   ) {
-    support = _support;
     votingToken = _votingToken;
     romulusDelegate = _romulusDelegate;
-
     _votingToken.delegate(address(this));
   }
 
-  function addVotes(uint256 amount) external onlyOwner {
+  /// @notice
```

Comment:
I think we should actually delete this function. It'll save gas because we won't need to call `approve` from `VotableStakingRewards`. Instead we can just call `safeTransfer` from the `VotableStakingRewards` contract